app/page-partials/my-dashboard/IntegrationList.tsx

```diff
@@ -3,7 +3,8 @@ import Link from '@button-inc/bcgov-theme/Link';
 import { Integration } from 'interfaces/Request';
 import padStart from 'lodash.padstart';
 import Grid from '@button-inc/bcgov-theme/Grid';
-import { Table, Button, NumberedContents, Header } from '@bcgov-sso/common-react-components';
+import { Button, NumberedContents, Header } from '@bcgov-sso/common-react-components';
+import Table from 'components/TableNew';
```

Comment:
naming?

Reply:
This is still using the inconsistent naming Table vs TableNew

Reply:
consistent with import Table from 'components/TableNew'; in all import table stmt